packages/jest-runtime/src/script_transformer.js

```diff
@@ -517,11 +520,25 @@ const calcIgnorePatternRegexp = (config: ProjectConfig): ?RegExp => {
   return new RegExp(config.transformIgnorePatterns.join('|'));
 };
 
-const wrap = content =>
-  '({"' +
-  ScriptTransformer.EVAL_RESULT_VARIABLE +
-  '":function(module,exports,require,__dirname,__filename,global,jest){' +
-  content +
-  '\n}});';
+const wrap = (content, ...extras) => {
+  const globals = new Set([
```

Comment:
Not sure it makes sense to wrap in a `Set` as that will mess up the order from `...extraGlobals.map(globalVariable => global[globalVariable]),` above

Reply:
Then I'll make sure both are wrapped in a `Set`, since if they pass a dupe by a mistake it'll throw over duplicate declaration.